packages/augur-state/src/Controller.ts

```diff
@@ -1,7 +1,7 @@
-import { Augur } from "@augurproject/api";
```

Comment:
At the risk of starting a holy war, doesn't the UI code have spaces here on import lines? Seems like we should be consistent.